src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseBackupLongTermRetentionPolicy.cs

```diff
@@ -41,20 +69,94 @@ public class SetAzureSqlDatabaseBackupLongTermRetentionPolicy : AzureSqlDatabase
         /// Gets or sets the name of the backup long term retention policy
         /// </summary>
         [Parameter(Mandatory = true,
+            ParameterSetName = LegacySet,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The Resource ID of the backup long term retention policy.")]
         [ValidateNotNullOrEmpty]
         [Alias("Id")]
         public string ResourceId { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not to clear the Long Term Retention V2 policy.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = ClearSet,
+            Position = 3,
+            HelpMessage = "If provided, the policy for the database will be cleared.")]
+        public SwitchParameter Clear { get; set; }
```

Comment:
I think -Clear might not be the best name... "-RemovePolicy" might be better. What do you think?

Reply:
Ping.

Reply:
Sounds good to me!